com.unity.postprocessing/PostProcessing/Runtime/Effects/AutoExposure.cs

```diff
@@ -121,12 +121,14 @@ void CheckTexture(int eye, int id)
 
         public override void Render(PostProcessRenderContext context)
         {
+            // multi pass and singlepass only has a single eye
+            int eye = (context.numberOfEyes>1) ? context.xrActiveEye : 0;
```

Comment:
The current eye is already passed into RenderBuiltins(context, ..., eye) before calling autoExposure.Render(), which we can use to determine whether in multipass or SPI mode. Can you try passing in current eye to autoExposure.Render() instead of calculating it here?

Reply:
FYI i was debating between changing the Render() signature to receive an eye or to store it in the context.

since xrActiveEye was already in the conext, it felt right to put it next to it so that if someone needs one eye and it's not the right one... he's not gonna have issue finding the other eye.

it also avoided modifying the signature of every PPFX